webapp/javascript/components/FlameGraphRenderer.jsx

```diff
@@ -492,6 +495,11 @@ class FlameGraphRenderer extends React.Component {
         key={'flamegraph-pane'}
         className={clsx("pane", { hidden: this.state.view === "table", "vertical-orientation": this.props.viewType === "double" })}
       >
+        <div className='flamegraph-header'>
+          <span></span>
+          <span>Frame width represents CPU time per function</span>
```

Comment:
I think this is great. However the text is a really bright white that is kind of distracting. Could we make it a little more grey maybe something like this: `#a7a7a7` And lets make the font size slightly smaller as well maybe ~`14px`